docs/docs/import-external-csv-files-to-landing-zone.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Importing external CSV files to the Data Platform Landing Zone
+description: "Overview of how CSV files from external suppliers are imported to the Data Platform Landing Zone"
+layout: playbook_js
+tags: [playbook]
+---
+
+This section covers the technical overview of how external suppliers can import data into the data platform via .csv files.
+
+The terraform script [33-liberator-iam.tf](https://github.com/LBHackney-IT/Data-Platform/tree/main/terraform/33-liberator-iam.tf) provisions an AWS IAM User and associated policies. The policies will allow the external supplier to see an S3 specified by the policy and to upload a file into that bucket and see the file. In the example script, this bucket is `s3://dataplatform-stg-landing-zone/parking/`  
```

Comment:
The S3 bucket specified in the policy is `"dataplatform-stg-liberator-data-storage/parking/*"`